pykafka/balancedconsumer.py

```diff
@@ -553,6 +553,8 @@ def consumer_timed_out():
             try:
                 message = self._consumer.consume(block=block)
             except ConsumerStoppedException:
+                if hasattr(self, '_running') and not self._running:
```

Comment:
Yes, it's an oversight on my part that `_running` isn't set to `False` on `__init__`. I think that's preferable to using `hasattr` in this case.